iothub/service/src/Microsoft.Azure.Devices.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net472;net451</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net472;net451;netcoreapp2.1</TargetFrameworks>
```

Comment:
As discussed in standup, in order to handle the use case when a user is on Net Core 2.1, we need to target that framework and add some preprocessor directives to use the SocketsHttpHandler as seen above

Reply:
worth adding a note here stating the reason why we added netcoreapp here (for future reference). Also, did you get a chance to talk to Chris about this?